{"uuid":"ece695ca-9fac-47d3-814d-3ce20a8c98cc","historyId":"a0dad9fe63f859b72e94f2868b739d01","fullName":"Pays.posfinaltesting.TakeOutTest.TakeOut_Card_FinalWayPayment","labels":[{"name":"package","value":"Pays.posfinaltesting.TakeOutTest"},{"name":"testClass","value":"Pays.posfinaltesting.TakeOutTest"},{"name":"testMethod","value":"TakeOut_Card_FinalWayPayment"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Card_Refund"},{"name":"subSuite","value":"Pays.posfinaltesting.TakeOutTest"},{"name":"host","value":"Pratik"},{"name":"thread","value":"13532@Pratik.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Takeout Card Final Way Payment"},{"name":"severity","value":"critical"}],"links":[{"name":"TC-026","type":"tms"}],"name":"TakeOut_Card_FinalWayPayment","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Card Transaction failed expected [true] but found [false]","trace":"java.lang.AssertionError: Card Transaction failed expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat Pays.posfinaltesting.TakeOutTest.TakeOut_Card_FinalWayPayment(TakeOutTest.java:360)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n"},"stage":"finished","description":"Perform Card Final Way Payment for Takeout Order","steps":[],"attachments":[{"name":"Screenshot on Failure","source":"3345914f-cebd-4c7c-a571-3ddc57622137-attachment"}],"parameters":[],"start":1742471402835,"stop":1742471414239}